src/constants/spacing.ts

```diff
@@ -1 +1 @@
-export const spacing = [4, 8, 16, 24, 32, 40, 56, 64, 104];
+export const spacing = [0, 4, 8, 16, 24, 32, 40, 56, 64, 104];
```

Comment:
I'm aware that this is part of the specs but we should be more granular here, as I'm pretty sure you're gonna be short a few values especially on the lower spectrum (similar to what tailwind offers in default config https://github.com/tailwindcss/tailwindcss/blob/master/stubs/defaultConfig.stub.js#L130). I'm saying this cause once we start using it, and then realize we're missing a few values in between, it's gonna be difficult to upgrade

Reply:
Good point 👍🏻  probably using an **Object** is gonna give us more flexibility in the future 💆🏻‍♂️

Reply:
Updated to use an object hope